docs/finops-alerts.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: FinOps Alerts
+browser: FinOps Alerts - Monitor your Azure environment for optimization
+nav_order: 40
+description: 'FinOps Alerts is your all-in-one solution for proactive cloud cost management.'
+permalink: /finops-alerts
+#customer intent: As a Finops practitioner, I need to learn about FinOps Alerts
+---
+
+<span class="fs-9 d-block mb-4">FinOps Alerts</span>
+Monitor your Azure environment for optimization.
+{: .fs-6 .fw-300 }
+
+<a class="btn btn-primary fs-5 mb-4 mb-md-0 mr-4" href="#deploy">Deploy</a>
+<a class="btn fs-5 mb-4 mb-md-0 mr-4" target="_blank" href="https://learn.microsoft.com/cloud-computing/finops/toolkit/finops-alert/overview">Documentation</a>
+
+---
+
+Designed with flexibility in mind, FinOps lens runs on a fully configurable schedule, continuously scanning your selected Azure subscriptions for idle resources.
+
+<div id="whats-new" class="m-0 p-4" style="background-color:#edf; border:solid 1px #609;">
+    <h3 class="m-0 mb-4">What's new in March 2025<span class="ftk-version">v0.8</span></h3>
+    <p class="mt-2 mb-0">
+        February released v1 of the solution.
+    </p>
+    <p class="mt-2 mb-0 ftk-externallink"><a target="_blank" href="https://learn.microsoft.com/cloud-computing/finops/toolkit/changelog">See all changes</a></p>
+</div>
+<a name="features"></a>
+
+## FinOps Alerts features
+
+<div class="ftk-gallery">
+    <div class="ftk-tile">
+        <div>💹 Idle resource monitoring</div>
+        <div>It scans selected subscriptions for idle resources that may increase costs.</div>
+    </div>
+    <div class="ftk-tile">
+        <div>📧 Automated notifications</div>
+        <div>Once idle resources are detected, the system sends alerts.</div>
+    </div>
+    <div class="ftk-tile">
+        <div>🔔 Customizable alerts</div>
+        <div>Users can customize email subjects, recipients, and subscription IDs to fine-tune the alerts.</div>
+    </div>
+    <div class="ftk-tile">
+        <div>📃 Review resource utilization</div>
+        <div>Track resource inventory and utilization of targeted services.<br>&nbsp;</div>
```

Comment:
Make sure this header doesn't wrap by default. Same with other sections, but we want all sections to be the same number of lines on a full-screen window. Only add the break and space if needed to align the sections.